src/pages/Contact.js

```diff
@@ -33,6 +34,10 @@ function Contact() {
     fetDataFromServer();
   }, []);
 
+  const handleModalClose = () => {
+    setIsModalOpen(false);
+  };
+
   const onSubmit = (data) => {
```

Comment:
Can we convert .then function to async await style of code

Reply:
changing it now!